metadata-io/src/test/java/com/linkedin/metadata/entity/DeleteReferencesServiceTest.java

```diff
@@ -0,0 +1,388 @@
+package com.linkedin.metadata.entity;
+
+import com.datahub.util.RecordUtils;
+import com.linkedin.data.DataList;
+import com.linkedin.data.DataMap;
+import com.linkedin.data.schema.DataSchema;
+import com.linkedin.data.schema.PathSpec;
+import com.linkedin.data.schema.grammar.PdlSchemaParser;
+import com.linkedin.data.schema.resolver.DefaultDataSchemaResolver;
+import com.linkedin.entity.Aspect;
+import com.linkedin.schema.SchemaMetadata;
+import junit.framework.TestCase;
+import org.testng.annotations.Test;
+
+public class DeleteReferencesServiceTest extends TestCase {
```

Comment:
Do we have tests for removing a required field? Even if we throw exception we should test that case for documentation

Reply:
ty!